examples/compose/docker-compose.yml

```diff
@@ -7,8 +7,10 @@ services:
   # created user-defined network and internal DNS for the name "consul".
   # Nodes will use Docker DNS for the service (passed in via the CONSUL
   # env var) to find each other and bootstrap the cluster.
+  # Note: Unless CONSUL_DEV is set, at least three instances are required for quorum.
   consul:
-    build: .
+    build: ../../
+    image: autopilotpattern/consul:${TAG:-latest}
```

Comment:
What is the point of both `build` and `image`? This file was more helpful without `image` and just building the container locally. I'm not sure what that is buying us.

Reply:
I was using the docker-compose file to build the image as well. I'll remove it.